src/Core/Silk.NET.SilkTouch/Diagnostics.cs

```diff
@@ -37,5 +37,17 @@ internal static class Diagnostics
             helpLinkUri: IssueLink,
             customTags: WellKnownDiagnosticTags.AnalyzerException
         );
+        
+        public static DiagnosticDescriptor SilkNetCoreMissing { get; } = new DiagnosticDescriptor
+        (
+            id: "ST0003",
```

Comment:
iirc in the other PR you've defined a conflicting ST0003

Reply:
Good Catch!